packages/@aws-cdk/aws-rds/test/instance.test.ts

```diff
@@ -251,6 +251,31 @@ describe('instance', () => {
 
   });
 
+  test('instance with inline parameter group', () => {
+    // WHEN
+    new rds.DatabaseInstance(stack, 'Database', {
+      engine: rds.DatabaseInstanceEngine.sqlServerEe({ version: rds.SqlServerEngineVersion.VER_12_00_5000_0_V1 }),
+      instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.SMALL),
+      vpc,
+      parameters: {
+        key: 'value',
+      },
+    });
+
+    expect(stack).toHaveResource('AWS::RDS::DBInstance', {
+      DBParameterGroupName: {
+        Ref: 'DatabaseParameterGroup2A921026',
+      },
+    });
+
+    expect(stack).toHaveResource('AWS::RDS::DBParameterGroup', {
+      Family: 'sqlserver-ee-12.0',
+      Parameters: {
+        key: 'value',
+      },
+    });
+  });
+
```

Comment:
Missing negative test (when both `parameters` and `parameterGroup` was provided).